test/stream.browser.spec.ts

```diff
@@ -66,3 +77,44 @@ describe('Stream Stats', () => {
     expect(datachannel.readyState).to.be.oneOf(['closing', 'closed'])
   })
 })
+
+describe('Stream Read Stats Transition By Incoming Flag', () => {
+  const webrtcStream = setup().webrtcStream
+  it('no flag, no transition', () => {
+    expect(webrtcStream.streamState.state).to.equal(underTest.StreamStates.OPEN)
+    const IncomingBuffer = generatePbByFlag()
+    const message = webrtcStream.processIncomingProtobuf(IncomingBuffer)
+    expect(message).not.equal(undefined)
+    ;(message != null) && expect(bytes.toString(message)).to.equal(TEST_MESSAGE)
```

Comment:
Is this intentional?

Reply:
not intentional ,changed by lint auto fix.

Reply:
original code is `message && expect(bytes.toString(message)).to.equal(TEST_MESSAGE)`

Reply:
It could have been related to a couple of things.  Can you put each `expect` on a separate line and remove the semicolon?

Reply:
We can remove the assertion on line 87, and only use `expect(bytes.toString(message)).to.eql(TEST_MESSAGE)`

Reply:
the type of message is `'Uint8Array | undefined'`, but `bytes.toString` need a Uint8Array param, so the no undefined chack should be left